CHANGELOG.md

```diff
@@ -21,6 +21,9 @@ _Please add entries here for your pull requests that are not yet released._
 #### Bug Fixes
 - Fix ReactDomClient initialization error during SSR. #1278
 
+#### Changed
+- The `react:component` generator now generates a function component by default #1271
+
```

Comment:
Shouldn't this PR go under "unreleased" rather then under 2.7.1, which was released last month?